src/schematools/types.py

```diff
@@ -286,7 +299,7 @@ def build_through_table(
         snakecased_field_id = to_snake_case(field.id)
         table_id = get_rel_table_identifier(len(self.id) + 1, table.id, snakecased_field_id)
 
-        sub_table_schema = {
+        sub_table_schema: Dict[str, Any] = {
```

Comment:
This is a recurring datatype. Maybe introduce an alias for it?

```python
Json = Dict[str, Any]
```

Or to provide some top level type checking until MyPy support recursive types:

```python
Json = Union[Dict[str, Any], List, str, int, float, bool, None, object]
```

Yes `Json` instead of `JSON`. Why? Loved this explanation in https://nim-lang.org/docs/nep1.html

> In the age of HTTP, HTML, FTP, TCP, IP, UTF, WWW it is foolish to pretend these are somewhat special words requiring all uppercase. Instead treat them as what they are: Real words. So it's parseUrl rather than parseURL, checkHttpHeader instead of checkHTTPHeader etc.

Reply:
Grappig, ik die ook altijd `checkHttpUrl` i.p.v. `checkHTTPURL`, maar als er niet geconcat wordt wel `URL` en dus `JSON` :-p. Hoewel ik wel weer een `QUrl` gebruik.

Reply:
I tried the approach with the `Json` alias. The first (simple) variation works. However, for the `Union`  type it is not just introducing the alias and doing a search-replace. This generates 19 mypy new errors. So this Union needs some more investigation.